ANMN/temperature_grid_product/README.md

```diff
@@ -0,0 +1,28 @@
+# anmn_temp_gridded_product.py
+
+This piece of Python 2.7 code creates a gridded temperature products based on FV01 NetCDF files for a particular deployment_code:
+1- It retrieves a list of FV01 NetCDF files for a particular deployment_code from the WFS service of view anmn_ts_timeseries_map.
+2- Checks that each file has the required variables, otherwise take them out of the list.
+3- Considers only data with flag 0, 1 or 2.
```

Comment:
only data with _quality control_ flag 0, 1, or 2, _meaning "not quality-controlled", "good data" or "probably good data"_
BTW, should it be including flag 0??

Reply:
> should it be including flag 0?

Yes, it was my intention. The idea is that I want to exclude "bad" data and not just include "god" data. So that for example we're still producing something if the data hasn't been fully QC'd (for example in/out water test flags bad anything outside deployment time period yet leaves QC flag to 0 for everything else).